src/main/java/nl/knaw/dans/ingest/core/service/mapper/mapping/DepositPropertiesOtherDoi.java

```diff
@@ -23,7 +23,7 @@
 public class DepositPropertiesOtherDoi extends Base {
 
     public static CompoundFieldGenerator<String> toOtherIdValue = (builder, value) -> {
-        builder.addSubfield(OTHER_ID_AGENCY, "");
-        builder.addSubfield(OTHER_ID_VALUE, value);
+        builder.addSubfield(OTHER_ID_AGENCY, value.replaceAll(":.*",""));
```

Comment:
It would be more robust to use split here. In theory the identifier could contain a colon. With greedy matching the wrong part would be replaced. For example: `PREF:some:id:value`.